<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RF_Paf</class>
 <widget class="QMainWindow" name="RF_Paf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RF_Paf</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="horizontalSpacing">
     <number>2</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="GB_comPorts">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="title">
       <string>COM Port</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="PB_start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_stop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CB_comList">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CB_baudList"/>
       </item>
       <item>
        <widget class="QPushButton" name="PB_refresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Parameters and results</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="L_powerMeasurement">
           <property name="text">
            <string>Power measurement [dBm]        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_powerMeasurement">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_powerMeasurement">
           <property name="text">
            <string>Measure</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="L_peakPower">
           <property name="text">
            <string>Peak power measurement:        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_peakPower">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_peakPower">
           <property name="text">
            <string>Measure</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="L_frequencyMeasurement">
           <property name="text">
            <string>Frequency measurement [MHz] </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_frequencyMeasurement">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_frequencyMeasurement">
           <property name="text">
            <string>Measure</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="S_frequencyMeasurement">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="L_maxPower">
           <property name="text">
            <string>Max power [dBm]                       </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LE_maxPower"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="PB_maxPower">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="L_kaliA">
           <property name="text">
            <string>A:                 </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_kaliA">
           <property name="text">
            <string>-40.114375</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="L_kaliB">
           <property name="text">
            <string>B:    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_kaliB">
           <property name="text">
            <string>24.995</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="L_kaliAtten">
           <property name="text">
            <string>Attenuation: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_kali_atten"/>
         </item>
         <item>
          <widget class="QLabel" name="L_kaliVcc">
           <property name="text">
            <string>Vcc: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_kaliVcc">
           <property name="text">
            <string>3.2812</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_kali">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="L_uPTemp">
           <property name="text">
            <string>uP temperature:                        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_uPTemp">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="L_detectorTemp">
           <property name="text">
            <string>Detector temperature:              </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_detectorTemp">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="L_refVoltage">
           <property name="text">
            <string>Reference voltage:                   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_refVoltage">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="L_serialNum">
           <property name="text">
            <string>Serial number:                           </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_serialnum">
           <property name="inputMask">
            <string>9999999999</string>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_serialNumSet">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_serialNumGet">
           <property name="text">
            <string>Read</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="9" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="L_firmVersion">
           <property name="text">
            <string>Firmware version:                      </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_firmVersion">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_firmVersion">
           <property name="text">
            <string>Read</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Error Logs</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="L_logs">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="L_errors">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
